app/controllers/dor_controller.rb

```diff
@@ -51,11 +51,15 @@ def reindex
     obj = Dor.load_instance params[:pid]
     @solr_doc = obj.to_solr
     Dor::SearchService.solr.add(@solr_doc, :add_attributes => {:commitWithin => 1000}) unless obj.nil?
-    index_logger.info "updated index for #{params[:pid]}"
+    message = "Successfully updated index for #{params[:pid]}"
+    index_logger.info message
+    flash[:notice] = message
+    redirect_to :back
   rescue ActiveFedora::ObjectNotFoundError # => e
-    index_logger.info "failed to update index for #{params[:pid]}, object not found in Fedora"
-    render :status => 500, :text => 'Object doesnt exist in Fedora.'
-    return
+    message = "Failed to update index for #{params[:pid]}, object not found in Fedora"
+    index_logger.info message
```

Comment:
maybe this log output should be `.warn` or `.error` call? @jmartin-sul how do we parse this log output, if at all?